src/modals/import-account-modal/import-account-seed/import-account-seed.form.tsx

```diff
@@ -38,10 +38,6 @@ export const importAccountSeedValidationSchema: SchemaOf<ImportAccountSeedValues
 
     const parts = p.replace('m', '').split('/').filter(Boolean);
 
-    if (parts.length < 4) {
-      return false;
-    }
-
```

Comment:
what if it's going to be `m` derivation path without any `/number'` is it correct?